lesson_13/maps_ts/src/lesson13.ts

```diff
@@ -3,5 +3,14 @@
  * https://leetcode.com/problems/permutation-difference-between-two-strings
  */
 export function findPermutationDifference(s: string, t: string): number {
-  return 0;
+  const bmb = new Map<string, number>();
```

Comment:
See earlier note on variable name.

Reply:
sounds good, that's on me that was a quick variable name I made I forgot to change it.